### PR TITLE
Organism Review Header : new fixture steps-animation

### PR DESCRIPTION
**Pull request by EmelineLeduc**
**Detailed purpose of the PR**
This PR includes the fixture steps-animation of Organism Review Header.
https://trello.com/c/qioPGb9m/2592-animate-the-transition-of-the-header-step-items-for-finished-revision

**Result and observation**

Tested with:

ie11
safari
chrome
mozilla

- chrome : 
Kapture 2022-05-09 at 15 22 20
Kapture 2022-05-09 at 15 24 41

Test in parent component : 
Kapture 2022-05-09 at 18 14 44

- ie : 
Kapture 2022-05-09 at 15 11 16
Kapture 2022-05-09 at 15 16 38

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by EmelineLeduc** [comment]
[Animate the transition of the header step items (for finished revision)](https://trello.com/c/qioPGb9m/2592-animate-the-transition-of-the-header-step-items-for-finished-revision)

**Review by jomaora — APPROVED** [review]

File: packages/@coorpacademy-components/src/organism/review-header/style.css [hunk 1]
```diff
@@ -108,6 +139,11 @@
     bottom: 20px;
   }
 
+  .stepsWrapperAnimation {
+    composes: stepsWrapper;
+    animation: stepsAnimation 0.5s ease-in-out 0.5s forwards;
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/organism/review-header/style.css:144]:
```suggestion
    opacity: 0;
    transform: opacity 0.5s ease-in-out 0.5s;
```

File: packages/@coorpacademy-components/src/organism/review-header/style.css [hunk 2]
```diff
@@ -47,6 +63,11 @@
   display: flex;
 }
 
+.stepsWrapperAnimation {
+  composes: stepsWrapper;
+  animation: stepsAnimation 0.5s ease-in-out 0.5s forwards;
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/organism/review-header/style.css:68]: pareil que pour l'autre, c'est possible avec transition

**Review by StefanoA1 — CHANGES_REQUESTED** [review]

**Comment by StefanoA1** [comment]
selon les gifs que je vois, la view mobile est cassé (tous les step items deviennent collés les uns avec les autres), normalement on attend ça:

Screenshot 2022-05-09 at 15 51 28

**Review by StefanoA1 — APPROVED** [review]

**Pull request merged**